taqueria-sdk/index.ts

```diff
@@ -100,6 +101,19 @@ export const getArch = (): LikeAPromise<string, TaqError> => {
 	}
 };
 
+export const getLatestFlextesaImage = async (arch: 'linux/arm64/v8' | 'linux/amd64'): Promise<string> => {
+	const response = await fetch('https://registry.hub.docker.com/v2/repositories/oxheadalpha/flextesa/tags');
+	const data = await response.json();
+	const tags = data.results;
+	for (const tag of tags) {
+		if (tag.name === 'latest') continue;
+		for (const image of tag.images) {
+			if (arch.includes(image.architecture)) return `oxheadalpha/flextesa:${tag.name}`;
+		}
+	}
+	return Promise.reject(new Error(`No flextesa image found for the platform of this device, which is ${arch}`));
+};
+
```

Comment:
Jimmy, I see that we're missing logic to ensure that this is the latest image?

Reply:
I don't think that's possible because the latest image changes overtime. And in this version, the `latest` only points to the "latest" for Intel chip.

Reply:
What I had in mind was a method like:
```
function getDockerImageTags(dockerImageName: string, filter: FilterFn): DockerTags[] where DockerTag is {name, arch} 
```

And then:
```
function getLatestDockerImageTag(dockerImageName: string, filter: FilterFn): DockerTag
```

FilterFn would be `(tag: DockerTag) => bool` that allows the caller to filter the list of tokens before returning the result. This might be necessary to distinguish rc/beta tags from stable ones.

Reply:
In this version of flextesa, the M1 version is `rc` and I don't think they have plans to make it `stable`. Maybe they forgot to change the name, so I don't think such distinction is meaningful.

Reply:
This is an outdated change btw